compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -629,6 +629,12 @@ def AMDAIESinkIntoCore :
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIESinkIntoCorePass()";
 }
 
+def AMDAIESplitLogicalObjFifos :
+  Pass<"iree-amdaie-split-logical-objectfifos", "ModuleOp"> {
+  let summary = "Pass to split L2 buffers to distribute on multiple shimTiles and memTiles.";
```

Comment:
I've added more descriptions of the pass.

Reply:
This is a good question. Well, it should work for matmul transpose as it is going through the same tiling strategy as matmul. Currently this pass does base on the assumption that the outer two dimensions of the L2 allocation are the number of cores for distribution. I think it might not work for conv ops as the outer two dimensions don't necessary connect to the core being used? We'll need to investigate how this can be used for conv ops as a separate thread in the future.